src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -68,8 +68,8 @@ function OptionRowLHN(props) {
         return null;
     }
 
-    const isMuted = optionItem.notificationPreference === CONST.REPORT.NOTIFICATION_PREFERENCE.MUTE;
-    if (isMuted && !props.isFocused && !optionItem.isPinned) {
+    const isHidden = optionItem.notificationPreference === CONST.REPORT.NOTIFICATION_PREFERENCE.HIDDEN;
+    if (isHidden && !props.isFocused && !optionItem.isPinned) {
```

Comment:
I think we can remove the check for isPinned because I don't believe we will not allow users to [pin hidden chats](https://github.com/Expensify/Expensify/issues/309277#:~:text=A%20join%20icon%20will%20replace%20the%20thread%20icon%20when%20the%20users%20notification%20preference%20is%20hidden.%20Clicking%20join%20will%20effectively%20do%20the%20same%20thing%20as%20commenting%20on%20the%20thread%2C%20switch%20the%20users%20notification%20preference%20to%20immediately.%20We%27ll%20need%20some%20help%20from%20%40Expensify/design%20on%20this%20one.) in the future. But if we want to keep this for now and remove it later I think that's okay too.

Reply:
i dont mind either, but if i remove the check here then i should remove the ability to pin hidden chats here too. If i remove the pin feature here i think it could be confusing from a UX perspective why this chat report is different/unpinnable, at least until "join" report is implemented. 

I think changing this later would be better IMO but i don't have a strong opinion on this. If you want me to remove the ability to pin in this PR for hidden chats and update this check i can do that too 🙂

Reply:
True, sounds good. I agree with you to remove it in the PR where the `join` button is implemented.